cypress/e2e/app.spec.cy.ts

```diff
@@ -0,0 +1,44 @@
+let credentials;
+
+describe('application serving', () => {
+  before(() => {
+    // SETUP THE LOGIN SCENARIO
+    cy.task('scenario:setup', 'login').then((creds) => {
+      credentials = creds;
+    });
+    cy.visit('localhost:8080');
```

Comment:
I think there's a way you can define a base url in cypress, and this can be replaced with simple `cy.visit('/')`

Reply:
Added that to the cypress.config.json file